README.md

```diff
@@ -8,10 +8,22 @@ Setting up
 
 1. Clone repository
 2. Install dependencies with `npm install`
-3. Start server with `bin/www` (or use Forever or similar)
+3. Start server with `bin/www` (or use Forever or similar), or [configure systemd to start it automatically](#Usage-with-systemd)
 4. Go to `localhost:3000/users/<vine-user-id>/atom` or 
    `localhost:3000/users/<vine-user-id>/rss`
 
+Usage with systemd
+------------------
+
+1. Globally install vinefeed with `npm install -g`
```

Comment:
I do not think so. Omitting the optional dependencies would require an explicit `--no-optional` switch, so the command given here already installs them by default.